qiskit_experiments/base_analysis.py

```diff
@@ -106,9 +106,10 @@ def run(
                 if "success" not in res:
                     res["success"] = True
                 analysis_result = DbAnalysisResultV1(result_data=res, **analysis_result_parameters)
+                if "chisq" in res:
```

Comment:
The existing experiments will need to be updated to use this label, at the moment they all populate `"reduced_chisq"`

Reply:
We could keep that if you prefer.

Reply:
I don't have a strong preference, either way is fine with me